lib/puppetlabs_spec_helper/tasks/fixtures.rb

```diff
@@ -4,6 +4,9 @@
 require 'open3'
 require 'json'
 
+# Top level namespace for spechelper.
```

Comment:
Or you can use the common Ruby style:
```ruby
module PuppetlabsSpecHelper
  module Tasks
    module FixtureHelpers
      # ...
    end
  end
end
```

Reply:
https://rubystyle.guide/#namespace-definition

Reply:
Thanks for the suggestion. 

If nested modules have to exist, I actually prefer this way too but was battling with rubocop at the time.

Have updated accordingly.